test/test_triggers.py

```diff
@@ -51,5 +51,18 @@ def test_last_modified(self):
         row = self.select('od_structure_part', obj_id)
         assert last_mod != row['last_modification'], "Last modification not set on update of child table (still {})".format(row['last_modification'])
 
+    def test_identifier(self):
+        row = {
+                'identifier': '',
+                'level': decimal.Decimal('50.000')
+        }
+
+        obj_id = self.insert_check('vw_cover', row)
+
+        row = self.select('od_structure_part', obj_id)
+
+        identifier = row['identifier']
+        assert identifier is not None, "Identifier not set on insert"
```

Comment:
Or even better `assert identifier, "Identifier not set on insert: '{}'".format(repr(identifier))`